package.json

```diff
@@ -1,13 +1,26 @@
 {
   "name": "linter-pylint",
   "linter-package": true,
-  "main": "./lib/init",
+  "main": "./lib/main",
   "version": "0.2.2",
   "description": "Lint python on the fly, using pylint",
   "repository": "https://github.com/AtomLinter/linter-pylint",
   "license": "MIT",
   "engines": {
-    "atom": ">0.50.0"
+    "atom": ">=1"
   },
-  "dependencies": {}
+  "providedServices": {
+    "linter": {
+      "versions": {
+        "1.0.0": "provideLinter"
+      }
+    }
+  },
+  "dependencies": {
+    "atom-linter": "^3.1.1",
+    "lodash": "^3.10.1",
+    "q": "^1.4.1",
+    "q-io": "^1.13.1",
+    "tmp": "0.0.26"
```

Comment:
helper module recently got a `tempFile` method by the way, You might wanna try that